manifests/apache.pp

```diff
@@ -53,10 +63,7 @@
         'params' => $api_proxy_params,
       },
     ],
-    'request_headers' => [
-      "unset ${remote_user_environ_header}",
-      "set ${remote_user_environ_header} \"%{SSL_CLIENT_S_DN_CN}s\" env=SSL_CLIENT_S_DN_CN",
-    ],
+    'request_headers' => concat($api_default_request_headers, $api_additional_request_headers),
```

Comment:
You can also write
```suggestion
    'request_headers' => $api_default_request_headers + $api_additional_request_headers,
```

Or keeping the diff smaller, you could keep the hardcoded array:
```suggestion
    'request_headers' => [
      "unset ${remote_user_environ_header}",
      "set ${remote_user_environ_header} \"%{SSL_CLIENT_S_DN_CN}s\" env=SSL_CLIENT_S_DN_CN",
    ] + api_additional_request_headers,
```

(Github butchers the diff)